code/modules/mob/living/carbon/human/human.dm

```diff
@@ -591,25 +591,25 @@
 		if(R && R.fields["criminal"])
 			switch(R.fields["criminal"])
 				if("*Arrest*")
-					threatcount += 5
+					return 1 // ordered to arrest
 				if("Incarcerated")
-					threatcount += 2
+					BAYES_THEOREM(threat_perc,0.99,0.05) // EXTREMELY likely scumbag, don't assume mistakes were made
 				if("Paroled")
-					threatcount += 2
+					BAYES_THEOREM(threat_perc,0.5,0.05) // watching you
 
 	//Check for dresscode violations
 	if(istype(head, /obj/item/clothing/head/wizard) || istype(head, /obj/item/clothing/head/helmet/space/hardsuit/wizard) || istype(head, /obj/item/clothing/head/helmet/space/hardsuit/shielded/wizard) || istype(head, /obj/item/clothing/head/helmet/space/hardsuit/syndi) || istype(head, /obj/item/clothing/head/helmet/space/hardsuit/shielded/syndi))
-		threatcount += 4 //fuk u antags <3			//no you
+		BAYES_THEOREM(threat_perc,0.7,0.04) // 70% they're wearing it if scumbag; 4% they're wearing it if not. Slightly lower because non-antags usually don't wear antag wear. Usually.
 
 	//mindshield implants imply trustworthyness
 	if(HAS_TRAIT(src, TRAIT_MINDSHIELD))
-		threatcount -= 1
+		BAYES_THEOREM(threat_perc,0.005,0.05) // mindshielded people HIGHLY unlikely to be scumbags
 
 	//Agent cards lower threatlevel.
 	if(istype(idcard, /obj/item/card/id/syndicate))
-		threatcount -= 2
+		BAYES_THEOREM(threat_perc,0.005,0.05) // fools 'em into thinking there's a mindshield, basically
```

Comment:
The reason its -2 is so you can mask + carry without sec bots going `Da fuck` I thought

Reply:
yes, this also does that